code/modules/projectiles/guns/energy/laser.dm

```diff
@@ -75,7 +75,7 @@
 	name = "xray laser gun"
 	desc = "A high-power laser gun capable of expelling concentrated xray blasts that pass through multiple soft targets and heavier materials"
 	icon_state = "xray"
-	item_state = "laser"
+	item_state = null
 	origin_tech = "combat=5;materials=3;magnets=2;syndicate=2"

```

Comment:
pretty sure it's intentional so there's *some* way of getting syndicate items besides it being a tator-tot round.

A lot of the augments require syndicate tech to get---as is it requires cargo or the experimentor to even make the X-Ray laserguns to begin with.